ec/ecdatasource/deploymenttemplates/model_elasticsearch.go

```diff
@@ -18,13 +18,14 @@
 package deploymenttemplates
 
 type elasticsearchModel struct {
-	HotTier          *topologyModel `tfsdk:"hot"`
-	CoordinatingTier *topologyModel `tfsdk:"coordinating"`
-	MasterTier       *topologyModel `tfsdk:"master"`
-	WarmTier         *topologyModel `tfsdk:"warm"`
-	ColdTier         *topologyModel `tfsdk:"cold"`
-	FrozenTier       *topologyModel `tfsdk:"frozen"`
-	MlTier           *topologyModel `tfsdk:"ml"`
+	AutoscalingEnabled *bool          `tfsdk:"autoscaling_enabled"`
```

Comment:
I think the corresponding `ec_deployement`'s attribute is called `autoscale`.
IMO it makes sense to use the same naming.

Reply:
Yeah true